var/spack/repos/builtin/packages/seacas/package.py

```diff
@@ -238,6 +244,17 @@ def cmake_args(self):
                 '-DTPL_ENABLE_CGNS:BOOL=OFF'
             ])
 
+        if '+faodel' in spec:
+            options.extend([
+                '-DTPL_ENABLE_Faodel:BOOL=ON',
+                '-DFaodel_ROOT:PATH=%s' % spec['faodel'].prefix,
+                '-DBOOST_ROOT:PATH=%s' % spec['boost'].prefix,
+            ])
+        else:
+            options.extend([
+                '-DTPL_ENABLE_Faodel:BOOL=OFF'
+            ])
+
```

Comment:
For safety (variant checking and dependency usage and more error-free code), use the CMakePackage helper macros:
```suggestion
        define = CMakePackage.define
        from_variant = self.define_from_variant

        options.append(from_variant('TPL_ENABLE_Faodel', 'faodel'))
        for pkg in ('Faodel', 'BOOST'):
            if pkg.lower() in spec:
                options.append(define(pkg + '_ROOT', spec[pkg.lower()].prefix]))

```

Reply:
@sethrj Thanks for the review.  I wasn't aware of these packages.  I have pushed updates to use them.